controller_manager/src/controller_manager.cpp

```diff
@@ -1263,6 +1274,41 @@ void ControllerManager::list_controllers_srv_cb(
         cs.required_state_interfaces = state_interface_config.names;
       }
     }
+    // check for chained interfaces
+    for (const auto & interface : cs.required_command_interfaces)
+    {
+      auto index = interface.find('/');
+      auto prefix = interface.substr(0, index);
+      if (controller_chain_map.find(prefix) != controller_chain_map.end())
+      {
+        controller_chain_map[cs.name].insert(prefix);
+        controller_chain_interface_map[cs.name].push_back(interface.substr(index+1, interface.size()-1));
+      }
+    }
+    auto references = controllers[i].c->export_reference_interfaces();
```

Comment:
@pac48 the error we discussed on Monday. Could it be caused with this call? You should check here is a controller is inactive. If so, then we should not list any interfaces whatsoever because they are probably not configured properly.

Reply:
I think as long as `on_init` is called, then the controller should know how to export its references, regardless of if its configured or active.

Reply:
I think the issue we was fixed with a clean build of the workspace.